cfgov/prepaid_agreements/jinja2tags.py

```diff
@@ -0,0 +1,36 @@
+from six.moves.urllib.parse import urlencode
+
+from django.utils.six import iterlists
+
+from jinja2.ext import Extension
+
+
+def remove_url_parameter(request, discards):
+    """
+    Removes specified params from request query
+    and returns updated querystring.
+    Discards should be a dict of lists:
+         {param: [values]}
+    """
+    query = request.GET.copy()
+    params = dict(iterlists(query))
+    for key in discards:
+        if key in params:
+            params[key] = [item for item in params[key]
+                           if item not in discards[key]]
+    querystring = urlencode(params, 'utf-8')
```

Comment:
This line causes a 500 due to a `UnicodeEncodeError` if the params contain a non-ASCII character, for example with URLs like:

- http://localhost:8000/prepaid-agreements/?q=debit&issuer_name=unicod%C3%AB
- http://localhost:8000/prepaid-agreements/?prepaid_type=Travel&search_field=all&q=unicod%C3%AB

This can be repeated by entering a search term with non-ASCII characters while filtering.

Reply:
@chosak Thanks for finding that issue! It should be fixed in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/60fbeca62dac8d65e736a0042c977a2c03d51193 (although the code may need some future cleanup).